project-koku/koku
# OCP on GCP: Include GCP database & network services on OCPALL tables.

Changes proposed in this PR:
* Add gcp specific database & network services to the ocp on all network & database tables.

Testing Instructions:
1. Add an ocp on gcp source, then check the following urls:
- http://localhost:8000/api/cost-management/v1/reports/openshift/infrastructures/all/costs/?group_by[service]=AmazonVPC,AmazonCloudFront,AmazonRoute53,AmazonAPIGateway,Virtual Network,VPN,DNS,Traffic%20Manager,ExpressRoute,Load%20Balancer,Application Gateway,Network,VPC,Firewall, Route,IP,DNS,CDN,NAT,Traffic Director, Service Discovery,Cloud Domains,Private Service Connect, Cloud Armor
- http://localhost:8000/api/cost-management/v1/reports/openshift/infrastructures/all/costs/?group_by[service]=AmazonRDS,AmazonDynamoDB,AmazonElastiCache,AmazonNeptune,AmazonRedshift,AmazonDocumentDB,Database,Cosmos DB,Cache for Redis,SQL,Spanner,Bigtable,Firestore,Firebase,Memorystore,MongoDB

These urls were not return ocp on gcp results before. 

---
Additional Context:
